content.yml

```diff
@@ -535,7 +535,7 @@ footer:
   neufund_txt: "NEUFUND"
   neufund_url: "https://neufund.org"
   commit_txt: "Commit"
-  commit_url: "https://neufund.org"
```

Comment:
hmm i think this should be just a link to the same page? `https://commit.neufund.org` ?